CoreInteractiveGraph/test/unit/src/au/gov/asd/tac/constellation/graph/interaction/gui/VisualGraphTopComponentNGTest.java

```diff
@@ -88,67 +89,69 @@ public static void tearDownClass() throws Exception {
     @Test
     public void testConstructor() {
         System.out.println("testConstructor");
-        Platform.runLater(() -> {
-            // Mock variables
-            final GraphDataObject mockGDO = mock(GraphDataObject.class);
-            final FileObject mockFileObject = mock(FileObject.class);
 
-            when(mockGDO.isInMemory()).thenReturn(false);
-            when(mockGDO.isValid()).thenReturn(true);
-            when(mockGDO.getPrimaryFile()).thenReturn(mockFileObject);
-            try {
-                when(mockGDO.createFromTemplate(any(), anyString())).thenReturn(mockGDO);
-            } catch (IOException e) {
-                LOGGER.log(Level.WARNING, "Caught exception in mockGDO", e);
-            }
+        // Mock variables
+        final GraphDataObject mockGDO = mock(GraphDataObject.class);
+        final FileObject mockFileObject = mock(FileObject.class);
 
-            when(mockFileObject.getPath()).thenReturn("");
+        when(mockGDO.isInMemory()).thenReturn(false);
+        when(mockGDO.isValid()).thenReturn(true);
+        when(mockGDO.getPrimaryFile()).thenReturn(mockFileObject);
 
-            // Assert constructed correctely
-            VisualGraphTopComponent instance = new VisualGraphTopComponent();
-            instance.getGraphNode().setDataObject(mockGDO);
-            assertEquals(instance.getGraphNode().getDataObject(), mockGDO);
-            assertNotNull(instance);
-            assertNotNull(instance.getUndoRedo());
-        });
+        try {
+            when(mockGDO.createFromTemplate(any(), anyString())).thenReturn(mockGDO);
+        } catch (IOException e) {
+            LOGGER.log(Level.WARNING, "Caught exception in mockGDO", e);
+            fail();
```

Comment:
may be better to simply throw the IOException in this case instead of catch if not really doing anything else in the catch (similar case in other unit tests)